specs/linux_file_system_hierarchy.md

```diff
@@ -168,10 +172,11 @@ Runtime system logs.
 System components may place private logs in this directory.
 Always writable, even when `/var/log/` might not be accessible yet.
 
-### `/run/opt/`
+### `/run/opt/` ⚠️
```

Comment:
what dir is this? this is completely made up, i cannot find any reference to it on the internet. 

It's one thing to be descriptive of stuff that is unnecessary/broken/bad but has tradition. but this dir seems like it is made up by the spec, and we shouldn't add new stuff to those legacy parts really.

Reply:
It's the corresponding of /etc/opt and /var/opt, but for runtime data, some 3rd party sw from microsoft uses it

Reply:
and /opt is optional, not legacy